### ISSUE TITLE
Remove BTC from meta information /mining

### ISSUE THREAD

**Issue by cuevasm** [blockstack/app.co#428]
Per Muneeb, we shouldn't be referencing Bitcoin if we don't have to. 

![image](https://user-images.githubusercontent.com/6131188/51059623-a98cb680-15a1-11e9-96ed-f55b3e95a0ea.png)

Suggest rewording to 'Earn cash payouts for..."

**Comment by aulneau** [comment]
I think 'cash' denotes fiat, which would be inaccurate. What about 'Earn payouts for apps...'?

**Comment by moxiegirl** [comment]
Could we just say "coin" without specifying the type?

**Comment by cuevasm** [comment]
Cash doesn't denote fiat, there even there's Bitcoin Cash. I chose it because the 'Cash for Code' language variation has outperformed anything else by a mile. If that's an issue, just put 'earn payouts' without being specific - it's more important we get this made quickly. Thanks

**Comment by pstan26** [comment]
I like this:

“Earn money by building apps people want. Blockstack is funding pioneering makers.”

If we get this resolved we can promote tweets directly to makers on twitter.
